src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,43 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 30px auto;
+  gap: $size-gap;
+  max-width: $max-width-calendar;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #eee;
+    height: $size-day;
+    width: $size-day;
+    border: 1px solid #000;
+    cursor: pointer;
+    transition-property: background-color, transform;
+    transition-duration: 0.5s;
```

Comment:
you can write this in one string, like `transition: background-color 0.5s, transform 0.5s;`

Reply:
Solved.